# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/HTMLEditor.h
# Commit: 3ea6eaf0dc16
# Full Hash: 3ea6eaf0dc166b86dfd68dd10738774991419795
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2022-08-04 09:46:07
# Regressor Bug: 1774704
# File Overlap Count: 1
# Description:
#   Bug 1774704 - part 9: Get rid of `EditorBase::TopLevelEditSubActionData::mNewBlockElement` due to unused r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152982
# ==============================================================================

diff -r 8896439912aa -r 3ea6eaf0dc16 editor/libeditor/HTMLEditor.h
--- a/editor/libeditor/HTMLEditor.h	Thu Aug 04 05:31:12 2022 +0000
+++ b/editor/libeditor/HTMLEditor.h	Thu Aug 04 05:39:15 2022 +0000
@@ -2639,8 +2639,6 @@
    * SetSelectionToAbsoluteAsSubAction() move selected contents to first
    * selected `<div>` element or newly created `<div>` element and make
    * the `<div>` element positioned absolutely.
-   * mNewBlockElement of TopLevelEditSubActionData will be set to the `<div>`
-   * element.
    *
    * @param aEditingHost        The editing host.
    */
